File: Sample/iOS_OSX/Common/LogModule.m
//
//  LogModule.m
//  Sample
//
//  Created by 冯鸿杰 on 16/9/22.
//  Copyright © 2016年 vimfung. All rights reserved.
//

#import "LogModule.h"

@implementation LogModule

- (void)writeLog:(NSString *)message
{
    NSLog(@"** message = %@", message);
}

@end

File: Sample/iOS_OSX/Sample-OSX-Swift/LogModule.swift
//
//  LogModule.swift
//  Sample
//
//  Created by 冯鸿杰 on 16/12/1.
//  Copyright © 2016年 vimfung. All rights reserved.
//

import LuaScriptCore_OSX_Swift

class LogModule: LSCModule
{
    static func writeLog(message : String) -> Void
    {
        NSLog("** message = %@", message);
    }
}
